### ISSUE TITLE
Foundation 6 Support

### ISSUE THREAD

**Issue by rolandschuetz** [million12/M12.Foundation#71]
Hello guys,

your plugins looks awesome. I love foundation and so I'm drawn to this a lot. Do you guys have any plans on building a Foundation 6 release? Any chance to work together?

Cheers,
Roland

**Comment by ryzy** [comment]
Hi Roland,

Yeah, we have plans... but nothing too solid yet. But I'm thinking more and more seriously about an update.

What about you, how much time would you be able to dedicate for it?

**Comment by rolandschuetz** [comment]
Hello,

I'm still learning Neos, so probably I need a few more weeks before I can be really of much help. I'm just finishing my first websites with Neos.

I can probably work on it 40-80 hours, so that should cover quite a big junk.

Bye the way, are you at Neos dev camp in two weeks?

**Comment by ryzy** [comment]
Roland, sounds groovy - let me know if you'd like to coordinate somehow. If I'm doing anything, I'll keep it posted here (and usually push my changes to another branch too). Unfortunately no plans for attending Neos camp/meetup anytime soon ;/

**Comment by ryzy** [comment]
FYI, I started work in [Pb.Site](https://github.com/million12/Pb.Site/tree/2.0) 2.0 branch (which is the "model" site package for M12.Foundation (old M12.FoundationSite)) and already migrated the build to Gulp4/Babel. This setup is basically a copy of Foundation 6 build. 

Some foundations for Foundation 6 are laid out... ;)

**Comment by RafaelKa** [comment]
Please create Issues with tasks and tag them with v6, i'll create PRs next Days.

**Comment by rolandschuetz** [comment]
Greeting from the dev camp.

Working with the project, I found that it actually does way to much to use it in production for a normal page. It would be cool though to have a plugin which does the basic foundation element and features and then a plugin on top which adds all the prototyping features.

Do you think that's a good idea?

**Comment by RafaelKa** [comment]
Yes, I realized it just by doing and switched to another approach ;). This is an overhead...
